### DISCUSSION TITLE
Cannot retry request with a non-repeatable request entity

### DISCUSSION THREAD

**Discussion by biosealjudielayson** [iterate-ch/cyberduck#16535] [Q&A] [answered]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS]

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.

**Additional context**

![image](https://github.com/user-attachments/assets/b386c94b-83c8-4faf-ae04-32cb90847516)

**Comment by AliveDevil** [comment, marked as answer]
Please don't use spaces in file names:
> Filenames are case-sensitive and should have no spaces. When creating a link in the steps below, ensure that the filename has no spaces and that the link matches the filename's letter case. You can rename the file after uploading it.

[Uploading and Linking to a File in Your Store](https://support.bigcommerce.com/s/article/How-do-I-add-and-link-to-a-file-in-my-store#upload-a-file)

**Comment by dkocher** [comment]
Also discussed in #16308.
